src/components/StudentFeedback.vue

```diff
@@ -1,21 +1,22 @@
 <template>
   <section class="feedback-container">
+    <router-view  :key="$route.path" :moduleId="$route.path"/>
     <div class="feedback-container__category">
-      <p class="feedback-container__category--label">{{this.categories[0]}} </p>
+      <p class="feedback-container__category--label">{{ categories[0] }} </p>
       <div class="progress-bar" id="bar1">3.5</div>
       <p class="feedback-container__category--comments">
         Solid work here! Blah blah.
       </p>
     </div>
     <div class="feedback-container__category">
-      <p class="feedback-container__category--label">{{this.categories[1]}} </p>
+      <p class="feedback-container__category--label">{{ categories[1] }} </p>
```

Comment:
I took out `this` because it's not needed here

Reply:
Ohhhhhhh thank you!!